packages/terra-embedded-content-consumer/src/EmbeddedContentProviderTestTemplate.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './ProviderIframe.module.scss';
+import './ProviderTestTemplate.scss';
+
+const propTypes = {
+  /**
+   * The content which is will embedded in an iframe by the XFC consumer.
```

Comment:
nit: Change `The content which is will embedded` to `The content which will be embedded`

Reply:
updated: https://github.com/cerner/terra-framework/pull/545/commits/b0627d752ac00b494b1ffe009102e267b4a1ad48